docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -609,39 +710,486 @@ Confirm Network Functionality
+        1    4d7b3a2e6411e5462d94208a5bb83b6c7652fa6f4c2ada1aa98cabb0be34af9d28cf3da0f8ccf414aac2230179becade7cdabbd0976c4846990f29e1f96000d6  1     1     034aad...
+        0    0fb3ebf6fdc5eef8af600eccc8d1aeb3d2488992e17c124b03083f3202e3e6b9182e78fef696f5a368844da2a81845df7c3ba4ad940cee5ca328e38a0f0e7aa0  3     11    034aad...
+
+   Block 0 is the :term:`genesis block`. The other two blocks contain
+   transactions for on-chain settings, such as setting PoET consensus.
+
+#. In a separate terminal window, connect to a different pod (such as pod 1) and
+   verify that it has joined the Sawtooth network.
+
+     .. code-block:: none
+
+        $ kubectl exec -it $(kubectl get pods | awk '/sawtooth-1/{print $1}') --container sawtooth-shell -- bash
+
+        root@sawtooth-1#
+
+   .. note::
+
+      The prompt ``root@sawtooth-1#`` marks the commands that should be run on
+      the Sawtooth node in pod 1.
+
+#. Display the list of blocks on the second pod.
+
+     .. code-block:: none
+
+        root@sawtooth-1# sawtooth block list
+
+   You should see the same list of blocks with the same block IDs, as in this
+   example:
+
+     .. code-block:: console
+
+        NUM  BLOCK_ID                                                                                                                          BATS  TXNS  SIGNER
+        2    f40b90d06b4a9074af2ab09e0187223da7466be75ec0f472f2edd5f22960d76e402e6c07c90b7816374891d698310dd25d9b88dce7dbcba8219d9f7c9cae1861  3     3     02e56e...
+        1    4d7b3a2e6411e5462d94208a5bb83b6c7652fa6f4c2ada1aa98cabb0be34af9d28cf3da0f8ccf414aac2230179becade7cdabbd0976c4846990f29e1f96000d6  1     1     034aad...
+        0    0fb3ebf6fdc5eef8af600eccc8d1aeb3d2488992e17c124b03083f3202e3e6b9182e78fef696f5a368844da2a81845df7c3ba4ad940cee5ca328e38a0f0e7aa0  3     11    034aad...
+
+#. (Optional) You can repeat the previous two steps on the other pods to verify
+   that they have the same block list. To connect to a different pod, replace
+   the `N` (in ``sawtooth-N``) in the following command with the pod number.
+   command:
+
+     .. code-block:: none
+
+        $ kubectl exec -it $(kubectl get pods | awk '/sawtooth-N/{print $1}') --container sawtooth-shell -- bash
+
+#. You can check the list of peers on the Sawtooth network. Run one or both of
+   the following commands from the shell container of any pod.
+
+   a. Use ``sawtooth peer list`` to show the peers of a particular node.
+
+      .. code-block:: console
+
+         root@sawtooth-N# sawtooth peer list
+         tcp://10.108.128.181:8800,tcp://10.108.70.26:8800,tcp://10.98.230.192:8800,tcp://10.98.230.192:8800,tcp://10.100.195.113:8800,tcp://10.100.195.113:8800,tcp://10.108.128.181:8800
+
+
+   b. Use ``sawnet peers list`` to display a complete graph of peers on the
+      network.
+
+      .. code-block:: console
+
+         root@sawtooth-N# sawnet peers list
```

Comment:
Currently, the ``sawnet peers list`` command doesn't work, because the Kubernetes multi-node environment is running Sawtooth 1.0.5 (not master).

Should I remove this command? Or should the yaml file have images that will run a more recent version of Sawtooth? (I.e., master at a certain point....)

Reply:
I resolved this issue with commit 4692495f64b3c..., which adds "(Sawtooth version 1.1 and later)" to the text in this step.
https://github.com/hyperledger/sawtooth-core/pull/1830/commits/4692495f64b3c699810041ca6d88c89f58c77063